server/routes/api/comments/comments.ts

```diff
@@ -360,4 +360,119 @@ router.post(
   }
 );
 
+router.post(
+  "comments.add_reaction",
+  rateLimiter(RateLimiterStrategy.TwentyFivePerMinute),
```

Comment:
Do we want it to increase it? Something like 60/sec? (Likewise for `remove_reaction` API below).

Reply:
Feels ok